### DISCUSSION TITLE
Output as HTML Table

### DISCUSSION THREAD

**Discussion by agent47nh** [wlezzar/jtab#3] [Ideas]
Adding HTML table output method would really be a great.

**Comment by wlezzar** [comment]
Hi @agent47nh : ) . Thanks for the idea. 

I just released a new version of jtab (v0.4.4) that supports the HTML output. I added some documentation about this feature in the Readme, it should be pretty straightforward to use: `jtab --format html > /tmp/table.html`.

Do not hesitate to give feedback : )

**Reply by agent47nh** [reply]
Thanks! Will give a feedback on it soon.
